msal/src/main/java/com/microsoft/identity/client/NativeAuthPublicClientApplication.kt

```diff
@@ -343,38 +323,53 @@ class NativeAuthPublicClientApplication(
                         )
                     )
                 }
-                is SignInCommandResult.InvalidCredentials -> {
+                is SignInCommandResult.Complete -> {
```

Comment:
We can merge the clauses for `Complete`, `InvalidCredentials` and `UnknownError` into `else` block. Or at least  `Complete`, `InvalidCredentials`  can be merged into a single else block.